src/routes.js

```diff
@@ -0,0 +1,2 @@
+export const toMovies = () => "/movies";
+export const toPeople = () => "/people";
```

Comment:
Dobry pomysł z osobnym plikiem dla routes!
Ale dlaczego w postaci funkcji? 
zwykłe zmienne:
```
export const toMovies =  "/movies";
export const toPeople = "/people";
```
zadziałają tak samo (oczywiście wtedy w App bez () ).

Reply:
Mi tak bardziej pasuje, bo wtedy będzie spójnie z kolejnymi ścieżkami, które będą wyświetlać strony dla konkretnych filmów i ludzi i będą musiały przyjąć w parametrze id, więc będą musiały być funkcjami.
Ale jak uważasz, że lepiej zmienne to mogę to zmienić :)

Reply:
Nie nie, chciałem właśnie tylko dowiedzieć się dlaczego. Dla mnie ok 👍